src/TrueLayer/Payouts/Model/Beneficiary.cs

```diff
@@ -15,29 +15,37 @@ public static class Beneficiary
         public sealed record PaymentSource : IDiscriminated
         {
             /// <summary>
-            /// Creates a new <see cref="PaymentSource"/>
+            /// Creates a new <see cref="PaymentSource"/>.
             /// </summary>
-            /// <param name="id">The TrueLayer merchant account identifier</param>
-            public PaymentSource(string id)
+            /// <param name="paymentSourceId">ID of the external account which has become a payment source.</param>
+            /// <param name="userId">ID of the owning user of the external account.</param>
+            /// <param name="reference">A reference for the payout.</param>
+            public PaymentSource(string paymentSourceId, string userId, string reference)
             {
-                Id = id.NotNullOrWhiteSpace(nameof(id));
+                PaymentSourceId = paymentSourceId.NotNullOrWhiteSpace(nameof(paymentSourceId));
+                UserId = userId.NotNullOrWhiteSpace(nameof(userId));
+                Reference = reference.NotNullOrWhiteSpace(nameof(reference));
             }
 
             /// <summary>
             /// Gets the type of beneficiary
             /// </summary>
-            public string Type => "merchant_account";
+            public string Type => "payment_source";
 
             /// <summary>
-            /// Gets the TrueLayer merchant account identifier
+            /// ID of the external account which has become a payment source.
             /// </summary>
-            public string Id { get; }
+            public string PaymentSourceId { get; }
```

Comment:
As this introduces a small breaking change, should we keep the old `Id` property and mark it as obsolete with a warning to use the new one? Something like:

```suggestion

            /// <summary>
            /// Gets the ID of the external account which has become a payment source
            /// Obsolete: please use <see cref="PaymentSourceId"/>
            /// </summary>
            [Obsolete("Please use PaymentSourceId instead")]
            public string Id { get; }
           
            /// <summary>
            /// Gets the ID of the external account which has become a payment source
            /// </summary>
            public string PaymentSourceId { get; }
```

Then in the constructor, you will simply do:
```
#pragma warning disable CS0618
                Id = PaymentSourceId;
#pragma warning restore CS0618
```

The `pragma` is to disable the warning about the property marked as obsolete.

In all fairness, I am not sure how many of our customers already use the payout functionality from the client (I know only one at the moment), so maybe a small breaking change would be ok in this case. It is always good thinking about introducing a gradual breaking change as best practice though 😉

Reply:
Apparently, one of the clients who tried to test the `payment_source` beneficiary said that it is not working. So in practice, we're breaking something that is already broken :D 

While you're generally right, on this specific occasion I think we can avoid putting that variable as obsolete. What do you think?

Reply:
That's what I thought initially but it was worth suggesting the change anyway 😄 
Happy with that